docs/content/latest/explore/query-1-performance/query-tuning-intro.md

```diff
@@ -0,0 +1,183 @@
+
+{{< /note >}}
+
+Once you identify the correct YSQL instance, YugabyteDB YSQL provides two views you can use to identify SQL statements and their performance characteristics:
+
+* `pg_stat_activity` - provides an overview, including SQL, of current activity. Apart from the query text, it does not provide information about the SQL executed; rather it shows the current state of the Postgres backend.
+
+* `pg_stat_statements` - provides historic information about executed SQL statements. Currently, no 'block'/'blk' information is recorded (which is logical and physical IO information), because the Postgres IO codepath is not used.
+
+Which view you use for performance tuning depends on the performance issue and whether you can witness the performance problem happening live, or if it was a performance problem in the past. 
+
+Note that identical queries might not perform identically because of differences in data.
+
+Once you identify a problematic SQL statement, the next step is to look at the execution plan. An execution plan is the order and method of execution to perform the work requested in the SQL statement. This plan is generated by the database planner.
+
+YugabyteDB uses PostgreSQL’s cost-based optimizer, which estimates the costs of each possible execution plan for an SQL statement. The planner calculates the cheapest plan to execute a query, assuming that is also the fastest and the best plan, based on statistics derived from the database table and supporting structures like secondary indexes. The execution plan with the lowest cost finally is executed. 
+
+{{< note title="Note" >}}
+
+Currently, YugabyteDB does not perform size-specific costing, and assumes every plan generates a thousand rows, which translates to a cost of a thousand. 
+
+{{< /note >}}
+
+To view the execution plan, use the `EXPLAIN` statement. You can use `EXPLAIN` in two modes:
+
+* The default `EXPLAIN` mode shows the plan that the planner created and the predicted cost.
+
+* `EXPLAIN ANALYZE` mode executes the query, measures the time and actual work done, and adds these statistics to the execution plan as 'actual'.
+
+Both the predicted and actual statistics have their uses: 
+
+* The predicted statistics are what the planner uses, and assuming the statistics don't change, the planner will come up with the same execution plan the next time the query is run, even if it caused performance issues in the past.
+
+* The actual statistics show how much and what work was actually performed, and thus are useful to see the effectiveness of the execution plan. 
+
+{{< tip title="Tip" >}}
+
+Complex execution plans can be difficult to read or understand. Use online tools to calculate and visualize execution plan timelines. For example, <https://explain.dalibo.com>.
+
+{{< /tip >}}
+
+## Performance tuning tools in YugabyteDB
+
```

Comment:
In general, try to avoid having two headings in a row, unless the introductory text you would put in between is really useless and contrived.